Package.swift

```diff
@@ -0,0 +1,39 @@
+// swift-tools-version:5.3
+import PackageDescription
+
+let package = Package(
+    name: "TreeSitterBash",
+    products: [
+        .library(name: "TreeSitterBash", targets: ["TreeSitterBash"]),
+    ],
+    dependencies: [],
+    targets: [
+        .target(name: "TreeSitterBash",
+                path: ".",
+                exclude: [
+                    "binding.gyp",
+                    "bindings",
+                    "Cargo.toml",
+                    "corpus",
+                    "grammar.js",
+                    "LICENSE",
+                    "Makefile",
+                    "package.json",
+                    "README.md",
+                    "script",
+                    ".travis.yml",
+                    "appveyor.yaml",
+                    "src/grammar.json",
+                    "src/node-types.json",
```

Comment:
I would guess the hesitation to merge this was because adding another list that needs to be updated whenever you add a file to the repo is not fun:

```suggestion
                    ".gitattributes",
                    ".github",
                    ".gitmodules",
                    ".npmignore",
                    "binding.gyp",
                    "bindings",
                    "Cargo.toml",
                    "corpus",
                    "examples",
                    "grammar.js",
                    "LICENSE",
                    "Makefile",
                    "package.json",
                    "README.md",
                    "script",
                    "appveyor.yaml",
                    "src/grammar.json",
                    "src/node-types.json",
```

Reply:
examples/ is the main one